docs/settings/task-manager-settings.asciidoc

```diff
@@ -29,7 +29,10 @@ Task Manager runs background tasks by polling for work on an interval.  You can
   | The maximum number of tasks that this Kibana instance will run simultaneously.  Defaults to 10.
     Starting in 8.0, it will not be possible to set the value greater than 100.
 
-  | `xpack.task_manager.monitored_stats_warn_delayed_task_start_in_seconds`
+  | `xpack.task_manager.monitored_stats_health_verbose_log.enabled`
```

Comment:
I think folks have been splitting this string up since it makes the column in the docs massive.

Reply:
Fixed!